libs/core/guards/animeStatus.ts

```diff
@@ -0,0 +1,9 @@
+import { AnimeStatus } from '../utils/types/anime';
+
+/**
+ * Checks if the value is a status.
+ * @param value Value, possibly being a status.
+ */
+export function isStatus(value: AnimeStatus | string): value is AnimeStatus {
```

Comment:
Wouldn't `string` be enough?
```suggestion
 * Checks if the value is a status.
 * @param value Value, possibly being a status.
 */
export function isStatus(value: string): value is AnimeStatus {
```